sdks/python/apache_beam/options/value_provider.py

```diff
@@ -75,7 +75,7 @@ def __hash__(self):
 
 class RuntimeValueProvider(ValueProvider):
   runtime_options = None
-  experiments = None
+  experiments = set()
```

Comment:
Would not it be better to do a `None` check in whereever this is used. If we initialize to set(), we cannot distinguish between an not yet initialized experiments from an actually an empty list.

Reply:
Is there any difference between an empty list of experiments and no initialization?